TestFlightFailure_IgnitionFail.cs

```diff
@@ -278,20 +278,34 @@ public override void SetActiveConfig(string alias)
             }
         }
 
-        public override string GetModuleInfo()
+        public override string GetModuleInfo(string configuration)
         {
             string infoString = "";
-            
-            if (baseIgnitionChance != null)
-            {
-                float pMin = baseIgnitionChance.Evaluate(baseIgnitionChance.minTime);
-                float pMax = baseIgnitionChance.Evaluate(baseIgnitionChance.maxTime);
-                infoString = $"Ignition chance at 0 data: <color=#859900ff>{pMin:P1}</color>\nIgnition chance at max data: <color=#859900ff>{pMax:P1}</color>";
-            }
 
-            if (pressureCurve != null & pressureCurve.Curve.keys.Length > 1)
+            foreach (var configNode in configs)
             {
-                infoString = $"{infoString}.\n<b>NOTE</b>: This engine suffers a penalty to ignition when air lighting due to dynamic pressure";
+                if (!configNode.HasValue("configuration"))
+                    continue;
+
+                var nodeConfiguration = configNode.GetValue("configuration");
+
+                if (string.Equals(nodeConfiguration, configuration, StringComparison.InvariantCultureIgnoreCase))
+                {
+                    if (configNode.HasNode("baseIgnitionChance"))
+                    {
+                        var nodeIgnitionChance = new FloatCurve();
+                        nodeIgnitionChance.Load(configNode.GetNode("baseIgnitionChance"));
+
+                        float pMin = nodeIgnitionChance.Evaluate(nodeIgnitionChance.minTime);
+                        float pMax = nodeIgnitionChance.Evaluate(nodeIgnitionChance.maxTime);
+                        infoString = $"  Ignition at 0 data: <color=#b1cc00ff>{pMin:P1}</color>\n  Ignition at max data: <color=#b1cc00ff>{pMax:P1}</color>";
+                    }
+
+                    //if (configNode.HasNode("pressureCurve"))
```

Comment:
Why is this commented out?

Reply:
To reduce the amount of text in the part catalogue info panels. It should really have been deleted rather than left lying around.